test/test.js

```diff
@@ -0,0 +1,17 @@
+var assert = require('assert');
+var greet = require('../greet.js');
+
+describe('Greet Module', function() {
+  describe('#greet()', function() {
+    it('returns Hello Jessica', function() {
+      assert.strictEqual('Hello Jessica', greet('Jessica'));
+    });
+    it('only works for strings', function() {
+      assert.strictEqual('Please enter a string', greet(9));
+    });
+    it('processes command line argument', function() {
+      process.argv[1] = 'some name';
```

Comment:
Nice job here! Mocking data by assigning it to the cli args is a solid way to test!